torch/nn/functional.py

```diff
@@ -4506,7 +4506,7 @@ def fold(
     tensor.
 
     .. warning::
-        Currently, only 3-D output tensors (unfolded batched image-like tensors) are
+        Currently, only 3-D (without batch dim) or 4-D (with batch dim) output tensors (unfolded image-like tensors) are
```

Comment:
How about this:
`Currently, only unbatched (3D) or batched (4D) image-like output tensors are supported.`